agent/engine/docker_task_engine.go

```diff
@@ -1336,28 +1336,21 @@ func (engine *DockerTaskEngine) createContainer(task *apitask.Task, container *a
 
 		containerCredSpec, err := container.GetCredentialSpec()
 		if err == nil && containerCredSpec != "" {
-			// CredentialSpec mapping: input := credentialspec:file://test.json, output := credentialspec=file://test.json
+			// on windows CredentialSpec mapping: input := credentialspec:file://test.json, output := credentialspec=file://test.json
+			// on linux CredentialSpec mapping: input := ssm/asm arn, output := /var/credentials-fetcher/krbdir/123456/ccname_webapp01_xyz
 			desiredCredSpecInjection, err := credSpecResource.GetTargetMapping(containerCredSpec)
 			if err != nil || desiredCredSpecInjection == "" {
-				missingErr := &apierrors.DockerClientConfigError{Msg: "unable to fetch valid credentialspec mapping"}
+				missingErr := &apierrors.DockerClientConfigError{Msg: "unable to fetch valid mapping associated with credentialspec"}
 				return dockerapi.DockerContainerMetadata{Error: apierrors.NamedError(missingErr)}
 			}
 
-			// Inject containers' hostConfig.SecurityOpt with the credentialspec resource
-			logger.Info("Injecting container with credentialspec", logger.Fields{
-				field.TaskID:     task.GetID(),
-				field.Container:  container.Name,
-				"credentialSpec": desiredCredSpecInjection,
+			logger.Info("Bind mount kerberos tickets with container", logger.Fields{
```

Comment:
this message is specific to Linux workflow right? but this code path would be hit for both Windows and Linux?